sdk/test-utils/test-credential/src/index.ts

```diff
@@ -2,10 +2,16 @@
 // Licensed under the MIT license.
 
 import {
-  DefaultAzureCredential,
+  AzureCliCredential,
+  AzureDeveloperCliCredential,
+  AzurePowerShellCredential,
+  ChainedTokenCredential,
   DefaultAzureCredentialClientIdOptions,
   DefaultAzureCredentialOptions,
   DefaultAzureCredentialResourceIdOptions,
+  EnvironmentCredential,
```

Comment:
`EnvironmentCredential` is useful if you haven't migrated yet.

Reply:
We'll keep it in for the packages that have not been migrated yet